packages/hash/frontend/src/components/hooks/blockProtocolFunctions/useBlockProtocolDeleteLinkedAggregations.ts

```diff
@@ -0,0 +1,67 @@
+import { useMutation } from "@apollo/client";
+
+import { BlockProtocolDeleteLinkedAggregationsFunction } from "blockprotocol";
+import { deleteLinkedAggregationMutation } from "@hashintel/hash-shared/queries/link.queries";
+import { useCallback } from "react";
+import {
+  DeleteLinkedAggregationMutation,
+  DeleteLinkedAggregationMutationVariables,
+} from "../../../graphql/apiTypes.gen";
+
+export const useBlockProtocolDeleteLinkedAggregations = (): {
+  deleteLinkedAggregations: BlockProtocolDeleteLinkedAggregationsFunction;
+  deleteLinkedAggregationsLoading: boolean;
+  deleteLinkedAggregationsError: any;
+} => {
+  const [
+    runDeleteLinkedAggregationsMutation,
+    {
+      loading: deleteLinkedAggregationsLoading,
+      error: deleteLinkedAggregationsError,
+    },
+  ] = useMutation<
+    DeleteLinkedAggregationMutation,
+    DeleteLinkedAggregationMutationVariables
+  >(deleteLinkedAggregationMutation);
+
+  const deleteLinkedAggregations: BlockProtocolDeleteLinkedAggregationsFunction =
+    useCallback(
+      async (actions) => {
+        const results: boolean[] = [];
+        // TODO: Support multiple actions in one GraphQL mutation for transaction integrity and better status reporting
+        for (const action of actions) {
+          if (!action.sourceAccountId) {
+            throw new Error("deleteLinks needs to be passed a sourceAccountId");
```

Comment:
```suggestion
            throw new Error("deleteLinkedAggregations needs to be passed a sourceAccountId");
```

Reply:
Thanks for spotting, fixed this (as well as another) error message in https://github.com/hashintel/hash/pull/497/commits/a3c5f693638883b246dd21c1a6dba35c3d4f70b5